### ISSUE TITLE
Missing `[scope]` attribute for table cells when changing their type

### ISSUE THREAD

**Issue by Comandeer** [ckeditor/ckeditor4#5084]
## Type of report

Feature request

## Provide description of the new feature

1. Open https://jsfiddle.net/Comandeer/obkd95u1/
2. Right click in the any of table cells and choose "Cell → Cell Properties".
3. Change "Cell Type" to "Header" and confirm the changes.

Cell is replaced with `th` element but this element lacks the `[scope]` attribute. I propose replacing the "Header" value in "Cell Type" with two other: "Column Header" and "Row Header". They should produce `th` elements with appropriate `[scope]` attributes.

**Comment by CKEditorBot** [comment]
Closed in https://github.com/ckeditor/ckeditor4/pull/5296
